delivery/consumer_test.go

```diff
@@ -9,6 +9,7 @@ import (
   // "strings"
   "regexp"
   "runtime"
+  // "fmt"
```

Comment:
Is this a typo?

Reply:
Not really. I find myself enabling and disabling fmt often while debugging, so I left it in commented to make it slightly easier to access next time I'm using it.